enterprise-suite/scripts/install-es.sh

```diff
@@ -40,14 +44,15 @@ function usage() {
     docvar ES_LOCAL_CHART "Set to location of local chart tarball"
     docvar ES_HELM_NAME "Helm release name"
     docvar ES_FORCE_INSTALL "Set to true to delete an existing install first, instead of upgrading"
+    docvar ES_EXPORT_FILE "Export resource yaml to this file.  Does not install in this case."
     docvar DRY_RUN "Set to true to dry run the install script"
     exit 1
 }
 
 # Create arg for the helm install/upgrade lines.  $1 is username.  $2 is password.
 # This prevents the credentials being written to a log.
 function set_credentials_arg() {
-    CREDS=$(mktemp -t creds.XXXXXX)
+    CREDS=$( mktemp -t creds.XXXXXX )
```

Comment:
How come?

Reply:
No reason.  A style thing I'm trying out...  You don't like?

Reply:
Yeah I prefer without spaces - it's more idiomatic.